src/main/java/org/opentripplanner/updater/bike_rental/GenericJsonBikeRentalDataSource.java

```diff
@@ -45,6 +48,9 @@
     /**
      * Construct superclass
      *
+     * @param severityFailureType The severity level to create an error with when fetching from this particular data
+     *                            source. This can be used to differentiate different error types for fetching from
```

Comment:
Suggestion: "This can be used to *define* different error types *when* fetching..."

Reply:
Refactored in 19d09a6